examples/large_models/deepspeed/bloom/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Loading large Huggingface models with constrained resources using accelerate
+
+This document briefs on serving large HG models with limited resource using deepspeed.
```

Comment:
by limited resource do you mean with CPU offloading? If so I'd mention that explicitly

Reply:
limited by one GPU device memory